ibtk/include/ibtk/LEInteractor.h

```diff
@@ -433,8 +485,8 @@ class LEInteractor
                             int Q_depth,
                             const std::vector<double>& X_data,
                             int X_depth,
-                            SAMRAI::tbox::Pointer<SAMRAI::pdat::SideData<NDIM, double> > mask_data,
                             SAMRAI::tbox::Pointer<SAMRAI::pdat::SideData<NDIM, double> > q_data,
+                            SAMRAI::tbox::Pointer<SAMRAI::pdat::SideData<NDIM, double> > mask_data,
```

Comment:
Did this just not work before?

Reply:
I had one-sided MLS for IBFE before  (that was removed from the library). @boyceg wants to revive it and we are doing it for IBMethod first.